doc/nrf/known_issues.rst

```diff
@@ -1130,6 +1130,13 @@ NCSDK-15675: Possible advertising start failure and module state error in :ref:`
 
   **Workaround:** Manually cherry-pick and apply commit with fix from main (commit hash: ``934a25ac23125758e350b64bca23885486682109``).
 
+.. rst-class:: v2-0-0
+
+NCSDK-15707: Due to changes in pwm drivers default pwm_period has been increased.
+  It causes visual glitches while LED color is updated for LEDs with more than one color channel.
```

Comment:
Maybe we could provide some context?
```
Update of the first LED channel occurs one PWM period earlier than update of remaining channels. Shorter LED PWM period mitigates the observed issue.
```

Reply:
Can we also add a short note about the problem in CAF LEDs documentation (I think we need it)